src/js/view/controls/controls.js

```diff
@@ -199,6 +199,17 @@ export default class Controls {
             api.setVolume(newVol);
         }
 
+        function onEscape() {
+            const related = api.getPlugin('related');
+            if (model.get('fullscreen')) {
+                api.setFullscreen(false);
+                this.playerContainer.blur();
+                this.userInactive();
+            } else if (related) {
```

Comment:
I prefer not having to create the `const` if we don't need to go down this code path. @karimJWP had a good suggestion of returning early, in which case we wouldn't need an else.

```
if() {
   ...
   return;
}
const related = api.getPlugin('related');
if (related) {   
   related.close({ type: 'escape' });
}

```